GitPullRequest.Services/GitRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using LibGit2Sharp;
+using LibGit2Sharp.Handlers;
+using Microsoft.Alm.Authentication;
+
+namespace GitPullRequest.Services
+{
+    public abstract class GitRepository
```

Comment:
I was wondering if `RemoteRepository` or `HostedRepository` would make sense for this class? `LibGi2Sharp.Repository` is also a "Git" repository, so the distinction isn't very obvious. GitHub and AzureDevOps are both hosts, so maybe `HostedRepository` would make more sense as the base class?